src/Symfony/Component/Polyfill/Tests/MbstringTest.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Polyfill\Tests;
+
+use Symfony\Component\Polyfill\Mbstring as p;
+use Normalizer as n;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ *
+ * @covers Symfony\Component\Polyfill\Mbstring::<!public>
+ * @requires extension mbstring
```

Comment:
well, I thought about that and I think it's better for tests to run against reference implementations (because we don't test the shim). So IMO it's better to keep the annotations.